### DISCUSSION TITLE
Discussions showing same comments over and over

### DISCUSSION THREAD

**Discussion by qw-in** [community/community#43776] [Mobile] [answered]
Got an update for a discussion I am following and the new comments just kept duplicating over and over as I scrolled down. Hitting the bottom it would continue to load in more duplicates

![Screenshot_2023-01-09-13-19-42-71.jpg](https://user-images.githubusercontent.com/19194187/211410808-87f0f2e6-a2bf-4d84-99df-59e08163df51.jpg)

Android 13

Edit: only happened the one time. Can not reproduce again

**Comment by eliperkins** [comment]
Hey @qw-in ! Thanks for this feedback. Can I get a little bit more information from you to help debug?

- What version of GitHub Mobile are you running?
- You mentioned it only happened once. Can you reproduce this on _other_ discussions with many comments? If you can reproduce this on other discussions that you haven't viewed yet, that might help us debug from our side.

**Comment by stevepopovich** [comment, marked as answer]
Hi @qw-in. We have a fix for this coming in the latest beta and next release. Thanks for reporting!
